### PR TITLE
style next pieces section

### LINKED ISSUE #35: "next pieces" styling
update the next-pieces component to match the designs

### LINKED ISSUE #46: Score stretches/shrinks the view
When score hits the board it seems to cause the board's width to expand

### LINKED ISSUE #47: Angular is different
Vue and React modes are identical but Angular is a little wider.

### PR DESCRIPTION
**Pull request by MattGiesa**
closes #35 
closes #47 
closes #46 

adds styling for next-pieces section
removes numbers from tiles

image

### CODE REVIEW

**Comment by MattGiesa** [comment]
this closes the "angular is different" issue because it sets a width to the sidebar, and angular was wider before because for whatever reason the sidebar was slightly wider in the angular version

**Comment by MattGiesa** [comment]
I accidentally left the settings file in (I wanted to disable Prettier for this project to avoid a bunch of formatting changes). I can figure out a way to remove it from the PR if anyone has a problem with it being committed

**Comment by smithad15** [comment]
I would remove the editor settings, please 🙂

File: src/front-end/angular/components/tile.component.ts [hunk 1]
```diff
@@ -4,11 +4,27 @@ import {
...
   selector: 'tile',
-  template: `{{ value }}`,
+  template: `
+  <style>
+    :host {
+      display: flex;
+      width: 100%;
+    }
+  </style>
+  <div [ngClass]="value === 0 ? emptyTile : tileByNumber(value)"></div>`,
```
Comment (MattGiesa) [inline src/front-end/angular/components/tile.component.ts]: I moved this logic into this component to be consistent with the other frameworks. I had to add the :host settings to keep the tile the right size. it's not ideal but it's easier to keep the frameworks all in sync this way

**Comment by MattGiesa** [comment]
removed the settings file

**Pull request merged**